tests/Functional/Fixtures/Symfony/AppCache.php

```diff
@@ -25,4 +29,14 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
 
         return $response;
     }
+
+    /**
+     * Made public to allow event subscribers to do refresh operations.
+     *
+     * {@inheritDoc}
+     */
+    public function fetch(Request $request, $catch = false)
+    {
+        return parent::fetch($request, $catch);
```

Comment:
i suspect that parent:: does not go to the trait but directly the parent. do we need to remap fetch from the trait to eventDispatchingFetch? or even change the trait method as it will never work when there is a name collision? looks like the most tricky bit of the trait, but don't see how we could avoid that.